src/beta_inc.jl

```diff
@@ -1012,21 +1006,15 @@ function _beta_inc_inv(a::Float64, b::Float64, p::Float64, q::Float64=1-p)
         if p_approx * p_approx_prev <= 0.0
             prev = max(sq, fpu)
         end
-        g = 1.0
 
-        tx = x - g*p_approx
-        while true
-            adj = g*p_approx
-            sq = adj^2
+	adj = p_approx
+	tx = x - adj
+	while prev <= (sq = adj^2) || tx < 0.0 || tx > 1.0
+            adj /= 3.0
             tx = x - adj
-            if (prev > sq && tx >= 0.0 && tx <= 1.0)
-                break
-            end
-            g /= 3.0
```

Comment:
I simplified this loop. Currently,
- the initial `adj = g*p_approx` and `tx` is computed twice
- `adj` is recomputed in every step as `g*p_approx` where `g` is divided by 3 in every iteration

In this PR, instead
- values in the initial iteration are only computed once
- `adj` is scaled directly, without variable `g` and additional multiplication